panel/config.py

```diff
@@ -834,6 +834,7 @@ def __call__(self, *args, **params):
         if not published:
             self._display_globals()
 
+    @classmethod
     def _display_globals(self):
```

Comment:
Shouldn't it be a `staticmethod` if it doesn't depend on `self` at all?

Reply:
I want to call it without initializing `pn.extension`, see https://github.com/holoviz/holoviews/pull/5947.

If I change to a `staticmethod` I get:
![image](https://github.com/holoviz/panel/assets/19758978/b62acf4f-6b8b-4290-a1b1-42fddbf0a630)

Reply:
Of course you need to remove `self` from the signature.

Reply:
This should be done.